src/content/en/updates/2018/12/badging-api.md

```diff
@@ -81,6 +81,14 @@ Examples of sites that may use this API include:
    running as an installed PWA (in your task bar or dock).
 3. Click the **Set** or **Clear** button to set or clear the badge from the app
    icon. You can also provide a number for the *Badge value*.
+   
+Note: While the Badging API *in Chrome* requires an installed app
+with an icon that actually can be badged, we advise against
+making calls to the Badging API dependent on the install state.
+The Badging API can apply to *anywhere* a browser might want to show a badge,
+so developers shouldn't make any assumptions about what situations
+the browser will make badges work. Just call the API when it exists.
+If it works, it works. If not, it simply doesn't.
```

Comment:
I think this is a little bit too intricate. If I understand this correctly, the intent is to just say that although the Badging API is available, when it is used, it's up to the browser to decide whether it's appropriate to show the badge or not?

Reply:
Again, after reading the pull request title, I realized that all you're trying to say is that the browser may show some form of badge even when the app is not installed. So, developers can call the Badging API (when available) even though their app is not installed...

Reply:
Yes. The badge API (ostensibly) has nothing to do with whether the app is installed or running in a window. It's just a way of saying "please associate this number with my website until further notice". The browser chooses when and how to show it; right now, we choose to show it on the app icon if it's installed, and we don't show it if it isn't installed.

Reply:
Thanks! I hope this feedback is dealt with in https://github.com/google/WebFundamentals/pull/7999/commits/9ec4212a74fbc2a285e9d162e1acada2852ed6e5.